example/rocketmq_v1alpha1_rocketmq_cluster.yaml

```diff
@@ -78,6 +92,12 @@ spec:
         items:
           - key: broker-common.conf
             path: broker-common.conf
+#          - key: plain_acl.yml
+#            path: plain_acl.yml
+#  volumeMounts:
+#    - name: broker-config
+#      subPath: plain_acl.yml
+#      mountPath: /root/rocketmq/broker/conf/plain_acl.yml
```

Comment:
IMO users should not care about the path of configuration files, operator should mount the correct path for it.

Reply:
I agree, how about support custom volume in this pr(#192), raise another pr to support #184  ?

Reply:
fixed.